<launch>

	<include file="$(find rcprg_gazebo_utils)/launch/gazebo_client.launch" />

	<include file="$(find rcprg_gazebo_utils)/launch/gazebo_publish_ros_tf_object.launch">
		<arg name="link_name" value="cafe_table::link" />
		<arg name="frame_id" value="table" />
	</include>

	<include file="$(find rcprg_gazebo_utils)/launch/gazebo_publish_ros_tf_object.launch">
		<arg name="link_name" value="cafe_table_0::link" />
		<arg name="frame_id" value="table_0" />
	</include>

	<include file="$(find rcprg_gazebo_utils)/launch/gazebo_publish_ros_tf_object.launch">
		<arg name="link_name" value="cafe_table_1::link" />
		<arg name="frame_id" value="table_1" />
	</include>

	<include file="$(find rcprg_gazebo_utils)/launch/gazebo_publish_ros_tf_object.launch">
		<arg name="link_name" value="cafe_table_2::link" />
		<arg name="frame_id" value="table_2" />
	</include>

	<include file="$(find rcprg_gazebo_utils)/launch/gazebo_publish_ros_tf_object.launch">
		<arg name="link_name" value="cafe_table_3::link" />
		<arg name="frame_id" value="table_3" />
	</include>

	<include file="$(find rcprg_gazebo_utils)/launch/gazebo_publish_ros_tf_object.launch">
		<arg name="link_name" value="cafe_table_4::link" />
		<arg name="frame_id" value="table_4" />
	</include>

</launch>